x-pack/plugins/alerting/server/lib/get_rule_task_timeout.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { RulesConfig } from '../config';
+
+const DEFAULT_EXECUTION_TIMEOUT = '5m';
+
+export const getRuleTaskTimeout = ({
+  config,
+  ruleTaskTimeout,
```

Comment:
**optional nit:** to make the naming of this variable clearer, what if it was called something like `ruleTypeProposedTimeout`?